src/components/calcite-label/calcite-label.tsx

```diff
@@ -4,6 +4,9 @@ import { FocusRequest } from "./interfaces";
 import { Alignment, Scale, Status } from "../interfaces";
 import { CSS_UTILITY } from "../../utils/resources";
 
+/**
+ * @slot - A slot for adding text.
```

Comment:
I think we should say text and a labelable component. We might need to document the labelable components here as well. thoughts @jcfranco ?

Reply:
@driskull do you mean a native label?

Reply:
No, one of our components that supports a calcite-label.